### PR TITLE
Fix #596: renaming "pagingState"/"nextPagingState" as "pageState"/"nextPageState"

### LINKED ISSUE #596: Pagination name discrepancy "nextPageState" vs "pagingState"
There is a slight discrepancy in the current name of the parameters around pagination:
- users receive `nextPageState`,
- but they must supply `pagingState` in their subsequent calls.

It's a small thing, yet it can be somewhat surprising and make things a tad less obvious from the user's perspective, which probably wonders why it is not consistently "paging" or "page" but half-and-half.

_(I see the only other issue referencing the name of these parameters is #260 which however is quite old and does not refer to this same point, so I opened a new issue here. Feel free to close if not considered relevant :) )_

### PR DESCRIPTION
**Pull request by tatu-at-datastax**
**What this PR does**:

Renames "pagingState" (and derived) as "pageState"

**Which issue(s) this PR fixes**:
Fixes #596

**Checklist**
- [ ] Changes manually tested
- [x] Automated Tests added/updated
- [ ] Documentation added/updated
- [x] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: src/test/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/DeleteOneCommandResolverTest.java [hunk 1]
```diff
@@ -65,7 +65,7 @@
 //                          assertThat(find.commandContext()).isEqualTo(commandContext);
```
Comment (Yuqi-Du) [inline src/test/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/DeleteOneCommandResolverTest.java:65]: why is this file commented out

File: src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/FindCommand.java [hunk 2]
```diff
@@ -46,12 +47,14 @@ public record Options(
...
 
-      // paging state for pagination
+      // page state for pagination
       @Schema(
               description = "Next page state for pagination.",
               type = SchemaType.STRING,
               implementation = String.class)
-          String pagingState,
+          @JsonProperty("pageState")
+          @JsonAlias("pageState")
```
[the remark this answers is not in the document]
Comment (tatu-at-datastax) [reply src/main/java/io/stargate/sgv2/jsonapi/api/model/command/impl/FindCommand.java:56]: And this is why we have code reviews! 100% yes, will change.

File: src/test/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/DeleteOneCommandResolverTest.java (hunk 1 above)
Comment (tatu-at-datastax) [reply src/test/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/DeleteOneCommandResolverTest.java:65]: I don't know, but looks like you commented it out? `git blame src/test/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/DeleteOneCommandResolverTest.java` shows:

```
52e3de288 (YuqiDu         2023-10-23 10:56:01 -0700 309) //                          assertThat(
52e3de288 (YuqiDu         2023-10-23 10:56:01 -0700 310) //                                  find.logicalExpression()
52e3de288 (YuqiDu         2023-10-23 10:56:01 -0700 311) //                                      .comparisonExpressions
...
```

Comment (tatu-at-datastax) [reply src/test/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/DeleteOneCommandResolverTest.java:65]: For:

```
commit 52e3de28808f2c3b756ac2c58e11233a0dee7838 (HEAD)
Author: YuqiDu <istimdu@gmail.com>
Date:   Mon Oct 23 10:56:01 2023 -0700

    Support and or (#565)
```

Maybe accidentally commented out?

Comment (Yuqi-Du) [reply src/test/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/DeleteOneCommandResolverTest.java]: sorry, fixed!

Comment (Yuqi-Du) [reply src/test/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/DeleteOneCommandResolverTest.java]: also fixed in cql branch

**Review by vkarpov15 — APPROVED** [review]
LGTM

**Review by Yuqi-Du — APPROVED** [review]

**Pull request merged**